athenahealth/appointments.go

```diff
@@ -622,6 +622,52 @@ func (h *HTTPClient) BookAppointment(ctx context.Context, patientID, apptID stri
 	return out[0], nil
 }
 
```

Comment:
We can add the success string as a private const outside of the `Update` function so that has some documentation around it (see comment below), something like this

```go
// UpdateBookedAppointment returns a 200 with a response of 1 if successful or
// an error string if not
const successfulBookingResponse string = "1"
```

Reply:
done! put it in generic.go just in case it's reusable